docs/TheThingsNetwork.md

```diff
@@ -174,3 +174,68 @@ void sleep(unsigned long mseconds);
 ```
 
 - `unsigned long mseconds`: number of milliseconds to sleep.
+
+# Comments
+## Serial ports (Stream objects)
+The Stream objects (Serial ports) need to be initialized at the correct baud rates at the start of your `setup()` function. See [our examples](https://github.com/TheThingsNetwork/arduino-device-lib/blob/asian-frequency-plans/examples) for more details. For example:
```

Comment:
Shouldn't we really start with the definition of the serials, so explain a bit about the hardware serial(s) and potential software serials?